README.md

```diff
@@ -72,15 +75,17 @@ A working example can be found in the `examples/` directory.
 // Also make sure the linker can find libimgui_xeno.a
 #[link(name = "imgui_xeno")] 
 extern "C" {
-    fn imgui_xeno_init(init: *const c_void, draw: *const c_void);
+    fn imgui_xeno_init(preInit: *const c_void, postInit: *const c_void, newFrame: *const c_void, draw: *const c_void);
```

Comment:
These names (and their usages/references in Rust files) should be snake case, i.e. `pre_init` etc.

Same for the exported functions in the same file.